deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,382 @@
+
+
+def get_script_arguments() -> argparse:
+    """Add and Receive the Script Parameters.
+
+    Returns:
+        argparse: Script parameters.
+    """
+
+    parser = argparse.ArgumentParser(
+        usage='%(prog)s [options]',
+        description='Script to Run Dashboard Saturation Tests',
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+
+    parser.add_argument(
+        '-l', '--log',
+        dest='logs',
+        type=str,
+        default='logs/',
+        help=f'Directory to store the logs. Default "logs".'
+    )
+
+    parser.add_argument(
+        '-s', '--screenshots',
+        dest='screenshots',
+        type=str,
+        default='screenshots/',
+        help=f'Directory to store the screenshots. Default "screenshot".')
+
+    parser.add_argument(
+        '-c', '--csv',
+        dest='csv',
+        type=str,
+        default='csv/',
+        help=f'Directory to store the CSVs. Default "csv".'
+    )
+
+    parser.add_argument(
+        '-o', '--session',
+        dest='session',
+        type=str,
+        default='.auth/',
+        help=f'Directory to store the Sessions. Default ".auth".'
+    )
+
+    parser.add_argument(
+        '-a', '--artillery',
+        dest='artillery',
+        type=str,
+        default="data/artillery.yml",
+        help=f'Path to the Artillery Script. Default "artillery.yml".'
+    )
+
+    parser.add_argument(
+        '-u', '--user',
+        dest='user',
+        type=str,
+        default='admin',
+        help=f'Wazuh User for the Dashboard. Default "admin".'
+    )
+
+    parser.add_argument(
+        '-p', '--password',
+        dest='password',
+        type=str,
+        required=True,
+        help=f'Wazuh Password for the Dashboard.'
+    )
+
+    parser.add_argument(
+        '-q', '--iterations',
+        dest='iterations',
+        type=int,
+        default=1,
+        help=f'Number of Tests to Run. Default {1}'
+    )
+
+    parser.add_argument(
+        '-i', '--ip',
+        dest='ip',
+        type=str,
+        required=True,
+        help=f'Set the Dashboard IP.'
+    )
+
+    parser.add_argument(
+        '-t', '--type',
+        dest='type',
+        type=str,
+        nargs='+',
+        action='store',
+        default=['aggregate', 'intermediate'],
+        help=f'JSON data to create the CSV.'
+    )
+
+    parser.add_argument(
+        '-w', '--wait',
+        dest='wait',
+        type=int,
+        default=5,
+        help=f'Waiting Time between Executions.'
+    )
+
+    parser.add_argument(
+        '-d', '--debug',
+        dest='debug',
+        action='store_true',
+        required=False,
+        default=False, help='Enable Debug Mode'
+    )
+
+    return parser.parse_args()
+
+
+def main() -> None:
+    """Run the Script.
+    """
+    script_args = get_script_arguments()
+
+    process_script_arguments(script_args)
+
+    for n in range(0, script_args.iterations):
+        run_artillery(script_args)
+        time.sleep(script_args.wait)
+
```

Comment:
```suggestion
    for _ in range(0, script_args.iterations):
        run_artillery(script_args)
        time.sleep(script_args.wait)

```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/a07ebb3ed73c7bc3ab8fce58bf98032b832575f8